packages/metalk8s-sosreport/src/metalk8s.py

```diff
@@ -0,0 +1,119 @@
+#! /bin/env python
+
+from sos.plugins import Plugin, RedHatPlugin, UbuntuPlugin
+from os import path
+
+
+class metalk8s(Plugin, RedHatPlugin, UbuntuPlugin):
+
+    """Metalk8s plugin"""
+
+    plugin_name = 'metalk8s'
+    packages = ('kubernetes', 'kubernetes-master')
+    profiles = ('container',)
+    files = ('/etc/kubernetes/admin.conf',)
+
+    option_list = [
+        ('all', 'also collect all namespaces output separately',
+            'slow', False),
+        ('describe', 'capture descriptions of all kube resources',
+            'fast', False),
+        ('podlogs', 'capture logs for pods', 'slow', False),
+    ]
+
+    def check_is_master(self):
+        return any([path.exists("/etc/kubernetes/admin.conf")])
+
+    def setup(self):
+        self.add_copy_spec('/etc/kubernetes')
```

Comment:
That includes sensitive data (private keys, certs, kubeconfigs...), not sure if we want them :/

Reply:
right, will make this precise

Reply:
okay everything here is sensitive, removing this

Reply:
maybe `/etc/kubernetes/manifests` could be useful, not sure though

Reply:
the manifests are not changed in runtime so they re already in the codebase

Reply:
I agree with @gdemonet we should include the manifest if for some reason a user edited the manifest we want to have them

Reply:
hmmm yes makes sens